pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -46,7 +46,7 @@ ENV TORCH_CUDA_ARCH_LIST="3.7 5.0 7.0+PTX 8.0"
 ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
 
 ENV CUDNN_VERSION=8.2.0.53
-ENV NCCL_VERSION=2.10.3
+ENV NCCL_VERSION=2.12.12
```

Comment:
Why are we modifying the base PyTorch container ? I see tests for base PyTorch container are disabled.